docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1900,6 +1900,86 @@ These are not currently modeled.
 #. Currently mailing addresses are the same as home addresses. This is not true, especially for rural populations. We plan to add this to the model later. 
 
 
+Social Security Observer
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Social security information is kept and tracked for a range 
+of different actions. For simplicity, we will limit this 
+section to only creation and dates of death. Others could be 
+added later, if desired. 
+
+**When to Sample** 
+
+- A sample of qualifying events is taken on every time step 
+
+**What to Sample** 
+
+.. list-table:: Simulant Attributes to Sample 
+  :widths: 20
+  :header-rows: 0
+
+  * - Unique simulant ID (for PRL tracking)
+  * - First name
+  * - Middle initial 
+  * - Last name
+  * - DOB 
+  * - Social Security Number 
+  * - Type of event 
+  * - Date of event 
+
+Currently, we will only track 2 types of events: 
+
+#. Creation of a SSN 
+#. Date of death recorded 
+
+The creation of a SSN can be triggered by a birth or by immigration into 
+the US. Both will be listed as "Creation" and are not differentiated. This 
+event is only recorded for simulates that receive a SSN. 
+The date of event is either the date of birth or the date of immigration. 
+If these values are unknown, any date within the time step can be recorded. 
+
+The date of death is a recording of anyone who has died and has a SSN. The 
+date of the event is the date of death. This will be listed as "Date 
+of Death". 
+
+Here is an example: 
+
+.. image:: SSA_example.png
+
+Note that the top row is a simulate that received a SSN at birth. The 
+third row indicates someone who immigrated at age 5 and received their 
+SSN then. 
+
+**Who to Sample** 
+
+100% of simulants with a SSN and a qualifying event in that time step 
+will be recorded. 
+
+**Data Errors/Noise** 
+
+- The omission rate (currently 0%) should be parameterizable. 
+- Additional noise in names and dates will be added in a separate function, at a later date. 
+
+**Limitations and Possible Future Adds** 
```

Comment:
I think this could be a bit confusing -- because "possible future adds" doesn't sound so different from saying that something is parameterizable.

I'd just say "Limitations," or if you want to keep the second part, somehow make it clear that this means in a future model, not the minimum viable model we will be using for our first US-scale run.

Reply:
Okay, makes sense. I changed the language to be "Items NOT Included in Minimum Viable Model" which I think is more clear.